app/src/main/java/org/systers/mentorship/view/activities/MainActivity.kt

```diff
@@ -97,34 +99,22 @@ class MainActivity: BaseActivity() {
 
         outState.putBoolean("atHome", atHome)
     }
-
-    private var doubleBackToExitPressedOnce = false
-
+    
     override fun onBackPressed() {
         if (!atHome) {
             showHomeFragment()
         } else if (atHome) {
-
             if (doubleBackToExitPressedOnce) {
                 super.onBackPressed()
                 return
             }
-
-
-
             doubleBackToExitPressedOnce = true
-
-
             val snack = Snackbar.make(getRootView(), "Press back again to exit", Snackbar.LENGTH_LONG)
             snack.setAction("EXIT",
                     View.OnClickListener {
```

Comment:
Use SAM syntax of Kotlin here, please. I'm not sure why you resolved the last conversation without any changing anything.